Gems/Multiplayer/Code/Include/Multiplayer/IMultiplayerDebug.h

```diff
@@ -24,5 +85,36 @@ namespace Multiplayer
 
         //! Disables printing of debug text over entities that have significant amount of traffic.
         virtual void HideEntityBandwidthDebugOverlay() = 0;
+
+        //! Adds a string based entry to the Multiplayer audit trail
+        virtual void AddAuditEntry(
+            const MultiplayerAuditCategory category,
+            const ClientInputId inputId,
+            const HostFrameId frameId,
+            const AZStd::string& name,
+            AZStd::vector<MultiplayerAuditingElement>&& entryDetails) = 0;
     };
+
+    #define AZ_MPAUDIT_INPUT_REWINDABLE(INPUT, REWINDABLE, VALUE_TYPE)                                                            \
```

Comment:
I would comment up these macros so everyone knows exactly what they do and what the parameters are.